tests/HLAPITestCase.php

```diff
@@ -190,13 +299,338 @@ public function call(Request $request, callable $fn, bool $auto_auth_header = tr
     {
         if ($auto_auth_header && $this->test->getCurrentBearerToken() !== null) {
             $request = $request->withHeader('Authorization', 'Bearer ' . $this->test->getCurrentBearerToken());
-        } else {
-            $this->router->registerAuthMiddleware(new InternalAuthMiddleware());
         }
         $response = $this->router->handleRequest($request);
         $fn(new HLAPICallAsserter($this->test, $this->router, $response));
         return $this;
     }
+
+    public function autoTestCRUD(string $endpoint, array $create_params = [], array $update_params = []): self
+    {
+        $this->test->resetSession();
```

Comment:
I haven't found the reason why, but suddenly the test user is already logged in somehow with a regular session before anything in a test is run. I went back two weeks in commits on `main` with the same results.